service/src/geopm_hint.h

```diff
@@ -17,14 +17,15 @@ extern "C" {
 enum geopm_region_hint_e {
     GEOPM_REGION_HINT_UNSET =     0ULL,       /* For clearing hints */
     GEOPM_REGION_HINT_UNKNOWN =   1ULL << 32, /* Region with unknown or varying characteristics */
-    GEOPM_REGION_HINT_COMPUTE =   1ULL << 33, /* Region dominated by compute */
-    GEOPM_REGION_HINT_MEMORY =    1ULL << 34, /* Region dominated by memory access */
-    GEOPM_REGION_HINT_NETWORK =   1ULL << 35, /* Region dominated by network traffic */
-    GEOPM_REGION_HINT_IO =        1ULL << 36, /* Region dominated by disk access */
-    GEOPM_REGION_HINT_SERIAL =    1ULL << 37, /* Single threaded region */
-    GEOPM_REGION_HINT_PARALLEL =  1ULL << 38, /* Region is threaded */
-    GEOPM_REGION_HINT_IGNORE =    1ULL << 39, /* Do not add region time to epoch */
-    GEOPM_REGION_HINT_INACTIVE =  1ULL << 40, /* Used to marke CPUs that are not running application */
+    GEOPM_REGION_HINT_COMPUTE =   2ULL << 32, /* Region dominated by compute */
```

Comment:
I think hint can actually range from 0-N (i.e. I don't think we need the 32 bit shift)?

Reply:
I opted to still do the shifting because it is still being embedded in region_id. See: ``Profile.cpp``. If the plan is to move away from that, I can look into it.